### ISSUE TITLE
Solving environment: failed  ResolvePackageNotFound when running conda env create --file text2mesh.yml

### ISSUE THREAD

**Issue by bibyts** [threedle/text2mesh#1]
When I run this command from the directory: conda env create --file text2mesh.yml

I get this error below. I have a CUDA GPU machine and it's enabled. I'm using Anaconda. How to fix?

==========================
Collecting package metadata (repodata.json): done
Solving environment: failed

ResolvePackageNotFound:
  - libxcb=1.14
  - dbus=1.13.18
  - libstdcxx-ng=9.3.0
  - gnutls=3.6.15
  - ld_impl_linux-64=2.35.1
  - libgomp=9.3.0
  - nettle=3.7.3
  - gstreamer=1.14.0
  - gst-plugins-base=1.14.0
  - gmp=6.2.1
  - libunistring=0.9.10
  - ncurses=6.2
  - _openmp_mutex=4.5
  - lame=3.100
  - openh264=2.1.0
  - readline=8.1
  - libuuid=1.0.3
  - libtasn1=4.16.0
  - libidn2=2.3.2
  - libgcc-ng=9.3.0

**Comment by ojmichel** [comment]
What version of CUDA are you using?

**Comment by bibyts** [comment]
CUDA Version: 11.4

This is what I get from system info when running numba -s

__CUDA Information__
CUDA Device Initialized                       : True
CUDA Driver Version                           : 11040
CUDA Runtime Version                       : 11010

**Comment by ojmichel** [comment]
A new yml file has been pushed. Can you try that one?

**Comment by bibyts** [comment]
Thanks! text2mesh installed but I'm stuck at figuring out how to run the shell script .sh from python command line in Anaconda. 

**These instructions: Run examples
Call the below shell scripts to generate example styles.**

I googled this and it sounds like I need to somehow create a subprocess to run shell script .sh file?

**Comment by bibyts** [comment]
Trying to run: run_alien_cobble.sh

# cobblestone alien
./demo/run_alien_cobble.sh

**Comment by ranahanocka** [comment]
Try running `bash ./demo/run_alien_cobble.sh` in a terminal

**Comment by bibyts** [comment]
I did that and got this error message:
./demo/run_alien_cobble.sh: line 1: python: command not found

**Comment by ranahanocka** [comment]
did you  activate conda env first? `conda activate text2mesh`

**Comment by bibyts** [comment]
Yes, but still getting that error for some reason when I run: bash ./demo/run_alien_cobble.sh

message: 
./demo/run_alien_cobble.sh: line 1: python: command not found

**Comment by factoryofthesun** [comment]
Can you show the output when you call `conda list python`? Can you also try deactivating and then activating the environment again? 
```
conda deactivate
conda activate text2mesh
```

**Comment by bibyts** [comment]
When I call conda list python:
[image: image.png]

**Comment by bibyts** [comment]
(text2mesh) C:\Users\Dan\text2mesh-main>conda list python
# packages in environment at C:\Users\Dan\anaconda3\envs\text2mesh:
#
# Name                    Version                   Build  Channel
ipython                   7.30.1           py37h03978a9_0    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
python                    3.7.12          h7840368_100_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python_abi                3.7                     2_cp37m    conda-forge

**Comment by factoryofthesun** [comment]
Have you tried the solutions here? https://stackoverflow.com/questions/36733179/why-conda-cannot-call-correct-python-version-after-activating-the-environment/57400535

You may also need to export your anaconda path to your environmental variables. https://stackoverflow.com/questions/49616399/windows-anaconda-python-is-not-recognized-as-an-internal-or-external-command

**Comment by bibyts** [comment]
Thanks. I will look into these. I got a Pip subprocess error when I originally ran:
conda env create --file text2mesh.yml

I think either my kaolin library isn't working right or my python installation is messed up. I did update the python system variables for Anaconda3.

Any recs for an environment to try this in other than Anaconda python terminal on Windows?

**Comment by factoryofthesun** [comment]
Yeah I'm not sure if you've tried just running the Python interpreter through your conda environment but my hunch is that something about your system build is causing problems. I've been able to run the code fine in Git Bash on Windows 10/11 so maybe try that?

**Comment by Pratik-Behera** [comment]
guys did you find solution for this? @factoryofthesun @bibyts

**Comment by factoryofthesun** [comment]
Are you experiencing a similar issue @Pratik-Behera? I haven't heard back from @bibyts.

**Comment by Pratik-Behera** [comment]
Yes still experiencing the same issue @bibyts
